### DISCUSSION TITLE
Scripts

### DISCUSSION THREAD

**Discussion by francoisschwarzentruber** [tableaunoir/tableaunoir#125] [Ideas]
I am thinking about adding small scripts in your boards. We may imagine a lot of things:
- moving magnets
- adding constraints between magnets
- draw interactive geometrical figures
- change the color depending on the cursor position
- make some computation
I also keep in mind the possibility of having magnets that may contain some code, like in [https://www-fourier.univ-grenoble-alpes.fr/%7eparisse/tableaunoirxcas/index.html]

**Comment by francoisschwarzentruber** [comment]
https://github.com/tableaunoir/documentation/blob/main/scripts.md

**Reply by tekhnee** [reply]
This is extraordinary and a provocation for the imagination!
